atlas/templates/Makefile.common.gotmpl

```diff
@@ -129,3 +130,30 @@ ifeq ($(WITH_GATEWAY), true)
 else
 	@echo "Your application doesn't have swagger file"
 endif
+
+#configuration for helm
+{{ if .WithHelm }}
+.PHONY: helm-lint
+helm-lint:
+	cd helm && helm lint -f $(CHART_NAME)/minikube.yaml $(CHART_NAME)
```

Comment:
instead of using `helm` directly we could use so that everyone will use a single version
 https://github.com/infobloxopen/helm